doc/users/github_stats.rst

```diff
@@ -3,138 +3,91 @@
 GitHub Stats
 ============
 
-GitHub stats for 2020/07/16 - 2020/08/13 (tag: v3.3.0)
+GitHub stats for 2020/08/14 - 2020/09/15 (tag: v3.3.1)
 
 These lists are automatically generated, and may be incomplete or contain duplicates.
 
-We closed 25 issues and merged 73 pull requests.
-The full list can be seen `on GitHub <https://github.com/matplotlib/matplotlib/milestone/56?closed=1>`__
+We closed 15 issues and merged 39 pull requests.
+The full list can be seen `on GitHub <https://github.com/matplotlib/matplotlib/milestone/57?closed=1>`__
 
-The following 17 authors contributed 131 commits.
+The following 14 authors contributed 61 commits.
 
 * Antony Lee
-* Ben Root
 * Bruno Beltran
 * David Stansby
+* David Young
 * Elliott Sales de Andrade
-* Isuru Fernando
-* jbhopkins
+* Greg Lucas
 * Jody Klymak
+* johnthagen
 * Jouni K. Seppänen
-* Lee Johnston
-* linchiwei123
-* Neilzon Viloria
+* Richard Sheridan
+* richardsheridan
```

Comment:
this is a dupe?

Reply:
This can be fixed, but the page it's on is automatically generated, so I wouldn't do it here.

Reply:
Not important either way...